packages/odyssey-react-mui/src/Autocomplete.tsx

```diff
@@ -292,6 +315,80 @@ const Autocomplete = <
       testId,
     ]
   );
+
+  const renderVirtualizedRow = (props: ListChildComponentProps) => {
+    const { data, index, style } = props;
```

Comment:
Why don't we use deconstruction here?

```suggestion
  const renderVirtualizedRow = ({ data, index, style }: ListChildComponentProps) => {
```

Reply:
Yep, good suggestion. I wasn't sure exactly what I needed from `props`, but now that I know, I'll simplify it as you suggested.